diff -urN flare_linux_v015_1/distribution/flare.desktop.in flare_linux_v015_1-patched/distribution/flare.desktop.in
--- flare_linux_v015_1/distribution/flare.desktop.in	2011-12-26 12:48:26.000000000 +1100
+++ flare_linux_v015_1-patched/distribution/flare.desktop.in	2011-12-27 23:09:51.133271136 +1100
@@ -12,6 +12,6 @@
 Comment[uk]=Ізометричний рольовий екшн (Action/RPG) для одного гравця
 TryExec=@FLARE_EXECUTABLE_PATH@
 Exec=@FLARE_EXECUTABLE_PATH@
-Categories=Game;RolePlaying;
-Icon=flare.svg
+Categories=Game;AdventureGame;
+Icon=flare
 Terminal=false
